src/openklant/templates/master.html

```diff
@@ -1,72 +1,33 @@
-                <li><a class="link link--muted" href="https://github.com/maykinmedia/open-klant">Code op Github</a></li>
-              </ul>
-            </div>
-            <div class="footer__item">
-              <h5>Overige</h5>
-              <ul class="list">
-                <li><a class="link link--muted" href="https://www.vngrealisatie.nl/ondersteuningsmiddelen/zaakgericht-werken">Zaakgericht werken volgen VNG</a></li>
-                <li><a class="link link--muted" href="https://www.vngrealisatie.nl/producten/api-standaarden-zaakgericht-werken">API's voor Zaakgericht werken</a></li>
-                <li><a class="link link--muted" href="https://commonground.nl/">Common Ground</a></li>
-              </ul>
-            </div>
-          </div>
-
-          {% if request.user.is_authenticated %}
-          <div class="footer__row">
-            <div class="footer__item">
-              <code>{{ settings.RELEASE }}</code>
-            </div>
-            <div class="footer__item footer__item--right">
-              <code>{{ settings.GIT_SHA|default:"" }}</code>
-            </div>
-          </div>
-          {% endif %}
-
-        </footer>
-      </div>
-    {% endblock footer %}
-
-    {% block extra_scripts %}{% endblock %}
-  </body>
+{% load static %}<!DOCTYPE html>
+<html lang="en" class="view {% block view_class %}view--{% if request.resolver_match.namespaces %}{{ request.resolver_match.namespaces|join:'-' }}-{% endif %}{{ request.resolver_match.url_name }}{% endblock %}">
+    <head>
+        <meta charset="utf-8">
+        <title>{% block title %}{% if title %}{{ title }}{% else %}{{ settings.SITE_TITLE }}{% endif %} | {{ settings.PROJECT_NAME }}{% endblock %}</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="{% static 'ico/favicon.svg' %}" sizes="any" rel="icon" type="image/svg+xml" />
+        <link href="{% static 'ico/favicon-96x96.png' %}" sizes="96x96" rel="icon" type="image/png" />
+        <link href="{% static 'ico/favicon-32x32.png' %}" sizes="32x32" rel="icon" type="image/png" />
+        <link href="{% static 'ico/favicon.png' %}" sizes="16x16" rel="icon" type="image/png" />
+        <link href="{% static 'ico/favicon.png' %}" rel="shortcut icon">
+        <link href="{% static 'bundles/openklant-css.css' %}" media="all" rel="stylesheet" />
+        {% block extra_css %}{% endblock %}
+    </head>
+
+    <body class="view__body">
+        {% include "includes/header.html" %}
+
+        <main class="view__main">
+            {% block content %}
+                Override <code>{% verbatim %}{% block content %}{% endverbatim %}</code>
+                to get some actual content.
+            {% endblock %}
+        </main>
+
+        {% include "includes/footer.html" %}
+
+        {% block scripts %}
+            <script src="{% static 'bundles/openklant-js.js' %}"></script>
+        {% endblock scripts %}
```

Comment:
do you even have any JS that needs to be included? https://github.com/maykinmedia/open-klant/tree/feature/add-django-common/src/openklant/js doesn't have anything meaningful, and possibly you can drop the whole frontend stack/webpack config even.